docs/source/installation/full_installation.rst

```diff
@@ -0,0 +1,162 @@
+Full Installation
+=================
+
+1. **Installing Extras for Specific Document Types**: 
+   If you're processing document types beyond the basics, you can install the necessary extras:
+
+   .. code-block:: bash
+
+      pip install "unstructured[docx,pptx]"
+
+   For a complete set of extras catering to every document type, use:
+
+   .. code-block:: bash
+
+      pip install "unstructured[all-docs]"
+
+2. **Note on Older Versions**:
+   For versions earlier than `unstructured<0.9.0`, the following installation pattern was recommended:
+
+   .. code-block:: bash
+
+      pip install "unstructured[local-inference]"
+
+   While "local-inference" remains supported in newer versions for backward compatibility, it might be deprecated in future releases. It's advisable to transition to the "all-docs" extra for comprehensive support.
+
+3. **Installing detectron2**:
+   If you intend to leverage custom models from the LayoutParser Model Zoo, you'll need `detectron2`. 
+   
+   Follow the guidelines `here <INSERT_LINK_HERE>`_ to install it.
```

Comment:
```suggestion
```
let's delete this section. nothing special has been needed for awhile with the onnx version of detectron2.